testproject-tools-integration/Packages/manifest.json

```diff
@@ -2,7 +2,7 @@
     "registry": "https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-candidates",
     "dependencies": {
         "com.unity.ide.rider": "3.0.7",
-        "com.unity.multiplayer.tools": "1.0.0-pre.5",
+        "com.unity.multiplayer.tools": "https://github.com/Unity-Technologies/com.unity.multiplayer.tools.git#cfb8fd305c3e46bb1251b40ec5f6cb621a267e7f",
```

Comment:
I believe we should use packages from the registry, not git repos.

something like:
```
"com.unity.multiplayer.tools": "1.0.0-pre.6",
```

Reply:
@MFatihMAR We are changing to using git references. This allows us to update the referenced version without publishing a version to the internal registry.

Reply:
it's a needless step that wastes developer time and causes disruption in our actual release process. what's your concern?

Reply:
to be clear - while frank's PR updates the test project to reference an actual release, we will also use this same strategy to reference unreleased work so that we can perform integration testing between packages during active development

Reply:
then are we supposed to remember updating `manifest.json` files every time we cut a new release? would you suggest making that an extra step of release process?

Reply:
we have a separate task for how to support version compatibility testing, but yes. and that's true whether we use a release version or a git hash. it's a good thing - we want to be explicit when we declare compatibility between NGO and a version of the tools package, regardless of whether it's a public release or an incremental changeset in our develop branch.

Reply:
is replacing git hash with released tools version in the `manifest.json` a part of tools release process or NGO release process? if so, is everybody aware of it?

Reply:
I believe it will be part of the Tools release process.
Tools would update the manifest to point to any commit-hash on develop/release branch depending on what they want to be tested (Tools makes releases from develop).
This PR actually points it to a commit-hash on develop `cfb8fd305c3e46bb1251b40ec5f6cb621a267e7f`

Reply:
if you're happy with pointing to git hashes and updating them to point towards release versions in the `manifest.json` when we cut a new release, I'm happy with it too @ashwinimurt — yes?

Reply:
Tagging @fluong6 to note this step in the release process docs.